nixos/modules/services/networking/cato-client.nix

```diff
@@ -34,8 +27,7 @@ in
 
       serviceConfig = {
         Type = "simple";
-        #User = "cato-client";
-        User = "root";
+        User = "root"; # Note: daemon runs as root, tools sticky to group
```

Comment:
If the daemon runs as root anyway, is there any reason making cato-clientd setgid?